pubspec.yaml

```diff
@@ -16,7 +16,7 @@ publish_to: "none" # Remove this line if you wish to publish to pub.dev
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 # In Windows, build-name is used as the major, minor, and patch parts
 # of the product and file versions while build-number is used as the build suffix.
-version: 1.0.15+1
```

Comment:
please raise version to 16

Reply:
I'm not sure what you mean by that in the more general sense. should it be done automatically?

Reply:
This is the applicative version so it cannot be done automatically. we should raise it when needed. 
The automation does add the commit hash so it's not so critical but this is a way we can distinguish our app version (and also display it to our users)